pytorch_lightning/loggers/tensorboard.py

```diff
@@ -57,6 +57,9 @@ class TensorBoardLogger(LightningLoggerBase):
             directory for existing versions, then automatically assigns the next available version.
             If it is a string then it is used as the run-specific subdirectory name,
             otherwise ``'version_${version}'`` is used.
+        sub_dir: Sub-directory to group TensorBoard logs. If a sub_dir argument is passed
+            then logs are saved in ``/save_dir/version/sub_dir/``. Defaults to ``None`` in which
```

Comment:
Wont it make more sense to save `/save_dir/sub_dir/version_{count}` ?

Reply:
Not really, the purpose of `sub_dir` is to provide a folder similar to `checkpoints` that groups all logs **within a run** together.
See an example here: 
```
save_dir
├── version_0
│   ├── checkpoints
│   │   ├── checkpoint_0.ckpt
│   │   └── checkpoint_1.ckpt
│   └── sub_dir
│       ├── log_0.log
│       └── log_1.log
└── version_1
    ├── checkpoints
    │   ├── checkpoint_0.ckpt
    │   └── checkpoint_1.ckpt
    └── sub_dir
        ├── log_0.log
        └── log_1.log
```

Reply:
Ok I get you. I thought it was to group several runs together.